src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -1197,9 +1197,10 @@ internal void TakeSnapshotWithExemplar(bool outputDelta)
     }
 
     /// <summary>
-    /// Denote that this MetricPoint is reclaimed.
+    /// This method sets the member object references of MetricPoint to `null`.
```

Comment:
number of object references?

Reply:
I used "**member** object references" here to denote any fields/ properties of `MetricPoint` struct that are reference types.